packages/components/src/Popover/_docs/Popover.stories.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react"
+import { Meta, StoryObj } from "@storybook/react"
+import { AddIcon, MeatballsIcon } from "~components/Icon"
+import { Popover as PopoverComponent, PopoverProps, usePopover } from "../index"
+
+const meta = {
+  title: "Components/Popover",
+  component: PopoverComponent,
+  args: {
+    children: undefined,
```

Comment:
This way we can still override the content in later stories if we want.

```suggestion
// Also change within the Wrapper to just use the spread args
    children: <>Popover body that explains something useful. <a href="/">Optional link</a></>,
```